client/components/Assignments/AssignmentPreviewContainer/_test.jsx

```diff
@@ -113,7 +113,7 @@ describe('<AssignmentPreviewContainer />', () => {
             expect(topTools.contains(
                 <span
                     className="priority-label priority-label--1 sd-list-item__inline-icon"
-                    data-sd-tooltip="Priority: High"
+                    data-sd-tooltip="Priority: {{ name }}"
```

Comment:
in tests gettext returns the string untouched?

Reply:
When bootstrapping tests we don't instantiate Superdesk (or angular services).
Instead we mock these interfaces required (https://github.com/superdesk/superdesk-planning/blob/develop/client/tests.js#L26).

We could improve this at a later date by getting using get-superdesk-api-implementation.tsx, or a mocked version thereof

Reply:
np, I was just checking